plugins/module_utils/utils.py

```diff
@@ -146,3 +146,32 @@ def check_if_all_elements_are_none(values):
     :return: True if all elements are None, False otherwise. -> boo
     """
     return all(value is None for value in values)
+
+
```

Comment:
I think it may be a good idea for our PRs to align on which general functions we are using. 
The PR from @shrsr may have the most complete example now. https://github.com/CiscoDevNet/ansible-mso/pull/615/files#diff-3409e22b7188780872a6a0c90a47c28744a04759f0f5d53c5e5dfd5587fed0b4

This also sets templateName/templateId fields in the returned MSO value along with complete reference objects. Another example can be found in [#614](https://github.com/CiscoDevNet/ansible-mso/pull/614/files#diff-3409e22b7188780872a6a0c90a47c28744a04759f0f5d53c5e5dfd5587fed0b4R297).